fury/ui.py

```diff
@@ -5240,3 +5240,387 @@ def _set_position(self, coords):
+
+            _offsetY += 50
+
+    def get_last_child(self, node):
+        """Get the previous node to newly added node
+        """
+        if len(node.children) > 0:
+            self.get_last_child(node.children[-1])
+        else:
+            self._last_child = node
+
+    @property
+    def color(self):
+        """ Returns the background color of the bullet point list.
+        """
+
+        return self.panel.color
+
+    @color.setter
+    def color(self, color):
+        """ Sets background color of the bullet point list.
+        Parameters
+        ----------
+        color : list of 3 floats.
+        """
+
+        self.panel.color = color
+
+    def left_button_pressed(self, i_ren, _obj, _sub_component):
+        click_pos = np.array(i_ren.event.position)
+        self._click_position = click_pos
+        i_ren.event.abort()
+
+    def left_button_dragged(self, i_ren, _obj, _sub_component):
+        click_position = np.array(i_ren.event.position)
+        change = click_position - self._click_position
+        self.panel.position += change
+        self._click_position = click_position
+        i_ren.force_render()
+
+
+class BulletPoint2D(UI):
+    """Create a Bullet Point from a node
+    Attributes
+    ----------
+    panel: :class: 'Panel2D'
+        Container to hold UI elements.
+    bullet: :class: 'Disk2D'
+        Displays the bullet before the text.
+    bullet_textbox: :class: 'TextBLock2D'
+        Displays the node as text after the bullet.
+    """
+
+    def __init__(self, node="", bullet_radius=5,
+                 indent=5, draggable=False, size=(300, 20),
+                 position=(0, 0), bg_color=(1, 1, 1),
+                 bg_opacity=1, bullet_color=(0, 0, 0),
+                 text_color=(0, 0, 0)):
+        """
+        Parameters
+        ----------
+        node: str
+            Text to display after the bullet
+        bullet_radius: int
+            Radius of the bullet
+        indent: int
+            Indentation of the bullet point
+        draggable: Bool
+            If the card should be draggable
+        size : (int, int)
+            Width and height of the pixels of this UI component.
+        position : (float, float)
+            Absolute coordinates (x, y) of the lower-left corner of the
+            UI component
+        bg_color: (float, float, float)
+            Background color of the bullet point
+        bg_opacity: float
+            Background opacity
+        bullet_color: (float, float, float)
+            Color of the bullet
+        text_color: (float, float, float)
+            Color of the text
+        """
+        self.content_size = size
+        self.bullet_radius = bullet_radius
+        self.indent = 2 * self.bullet_radius + indent
+        self.draggable = draggable
+        self.bg_color = bg_color
+        self.bg_opacity = bg_opacity
+        self.bullet_color = bullet_color
+        self.text_color = text_color
```

Comment:
You should add these after calling the super constructor and implement the respective getter and setter methods

Reply:
Instead of these variables can I do something like this:-
```python
        super(BulletPoint2D, self).__init__()
        self.panel.color = bg_color
        self.panel.opacity = bg_opacity
        self.bullet.color = bullet_color
        self.bullet_textbox.color = text_color
        self.position = position
```